Automattic/newspack-plugin
# Campaigns Wizard: display campaign ID

### All Submissions:






### Changes proposed in this Pull Request:



https://github.com/Automattic/newspack-popups/pull/321 makes it possible to insert campaigns using a shortcode. To make it easier, the wizard will display a campaign ID (which is to be inserted into a shortcode)

### How to test the changes in this Pull Request:

1. Visit the Campaigns Wizard, observe that on the options popover visible after clicking the three dots icon, Campaign ID is shown 

### Other information:
